### DISCUSSION TITLE
Bruno integration with ReportPortal

### DISCUSSION THREAD

**Discussion by Estegar** [usebruno/bruno#1438] [Ideas]
Hi, there's https://github.com/reportportal/client-javascript and I think it would be nice to have integration with this. 
Postman already has integration with ReportPortal, so i think Bruno is a natural candidate here.
What do you think?

**Comment by bmplcm** [comment]
you can already leverage the junit xml format to import data into report portal.
in command line, if you use "--output results.xml --format junit", you'll get the junit xml format that you can leverage to ingest results into report portal

**Reply by Estegar** [reply]
Thanks for the suggestion, i will try that out!
